lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -4580,6 +4580,9 @@ LogicalResult FIRRTLLowering::lowerVerificationStatement(
       case EventControl::AtNegEdge:
         event = circt::sv::EventControl::AtNegEdge;
         break;
+      default:
+        event = circt::sv::EventControl::AtPosEdge;
+        break;
```

Comment:
```suggestion
      default:
        llvm::report_fatal_error("Invalid event");
```

Does this remove the warning? I think compile failure is better than implicitly rolling back to a random default value.